tests/rule_based_profiler/parameter_builder/test_regex_pattern_string_parameter_builder.py

```diff
@@ -0,0 +1,233 @@
+from typing import Set
+
+import pytest
+
+import great_expectations.exceptions.exceptions as ge_exceptions
+from great_expectations.data_context import DataContext
+from great_expectations.execution_engine.execution_engine import MetricDomainTypes
+from great_expectations.rule_based_profiler.parameter_builder import (
+    RegexPatternStringParameterBuilder,
+    regex_pattern_string_parameter_builder,
+)
+from great_expectations.rule_based_profiler.types import (
+    Domain,
+    ParameterContainer,
+    get_parameter_value_by_fully_qualified_parameter_name,
+)
+
+
+def test_regex_pattern_string_parameter_builder_instantiation():
```

Comment:
Is this test meant to always have parity with the default `CANDIDATE_REGEX`?

We should probably test:
1. One case where we use `CANDIDATE_REGEX`
2. Another case where we override with our own patterns

By creating parity between the list noted herein and the default on the class, we'll need to modify this test each time we want to update the `CANDIDATE_REGEX`. Are we okay with that?

Reply:
Thank you so much for this. Regarding the additional test, I just added it to the PR (thank you). Regarding your question as to whether a test that tracks our default strings should exist, I want to say "yes".  Since it's a longer/difficult to read list, I think it would be great to keep track of any inadvertent changes.